flags.md

```diff
@@ -0,0 +1,141 @@
+```nim
+
+## From build.nims
+
+# Arch:
+
+--cpu:arm                             # tags: arm32
+--cpu:arm64                           # tags: arm64
+--cpu:i386                            # tags: x86
+
+--define:bit32                        # tags: arm32
+--define:bit32                        # tags: x86
+
+--gcc.path:/usr/bin                   # tags: arm64
+--gcc.exe:aarch64-linux-gnu-gcc       # tags: arm64
+--gcc.linkerexe:aarch64-linux-gnu-gcc # tags: arm64
+
+--passC:'-m32'                        # tags: x86, gcc
+--passL:'-m32'                        # tags: x86, gcc
+
+# Developer
+
+--embedsrc:on                         # tags: dev
+--define:DEV                          # tags: dev
+--listCmd                             # tags: dev
+
+--define:DEBUG                        # tags: debug
+--debugger:on                         # tags: debug
+--debuginfo                           # tags: debug
+--linedir:on                          # tags: debug
+
+--define:PROFILE                      # tags: memprofile
+--profiler:off                        # tags: memprofile
+--stackTrace:on                       # tags: memprofile
+--define:memProfiler                  # tags: memprofile
+
+--define:PROFILE                      # tags: profiler
+--profiler:on                         # tags: profiler
+--stackTrace:on                       # tags: profiler
+
+--debugger:native                     # tags: profilenative
```

Comment:
## Current implementation

Questions:
* Why is `--profiler` is `off` for `memprofile`?
* What is the difference between the three approachs?

---

## New implementation

Questions:
* Should we add a `--profile` flag with the following options: `none`, `mem`, `default`, `native`?
* Should `--profile` be the default settings for `--who dev`? 
  * If yes, which of the options above should be the default one?